File: frontend/update_instagram.py
import glob, os

files = glob.glob(r'e:\pkrfiles\events-manager\frontend\css\*.html')
for file in files:
    with open(file, 'r', encoding='utf-8') as f:
        content = f.read()
    
    new_content = content.replace(
        '<a href="#" class="social-link">',
        '<a href="https://www.instagram.com/3ventmanager?igsh=MW10dDlwaGF5MXByYg==" class="social-link" target="_blank" rel="noopener noreferrer">'
    )
    
    if new_content != content:
        with open(file, 'w', encoding='utf-8') as f:
            f.write(new_content)
        print(f'Updated {file}')

File: frontend/update_instagram.js
const fs = require('fs');

const dir = 'e:/pkrfiles/events-manager/frontend/css/';
const files = fs.readdirSync(dir).filter(f => f.endsWith('.html'));

files.forEach(file => {
    const filePath = dir + file;
    const content = fs.readFileSync(filePath, 'utf-8');
    const newContent = content.replace(
        '<a href="#" class="social-link">',
        '<a href="https://www.instagram.com/3ventmanager?igsh=MW10dDlwaGF5MXByYg==" class="social-link" target="_blank" rel="noopener noreferrer">'
    );
    if (newContent !== content) {
        fs.writeFileSync(filePath, newContent, 'utf-8');
        console.log('Updated', filePath);
    }
});
